docs/book/src/component-config-tutorial/testdata/project/README.md

```diff
@@ -5,73 +5,70 @@
 // TODO(user): An in-depth paragraph about your project and overview of use
 
 ## Getting Started
-You’ll need a Kubernetes cluster to run against. You can use [KIND](https://sigs.k8s.io/kind) to get a local cluster for testing, or run against a remote cluster.
-**Note:** Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster `kubectl cluster-info` shows).
 
-### Running on the cluster
-1. Install Instances of Custom Resources:
+### Prerequisites
+- go version v1.20.0+
+- docker version 17.03+.
+- kubectl version v1.11.3+.
+- Access to a Kubernetes v1.11.3+ cluster.
```

Comment:
```suggestion
- Access to a Kubernetes v1.11.3+ cluster
```

Reply:
I am OK with this one. But you lgtm so, wdyt about doing that in a follow-up?
Could you raise a follow-up with this change?